readme.md

```diff
@@ -227,7 +227,9 @@ While annotations are not specified by IIIF they are an important enabling techn
 Content Management Systems (CMS) modules that implement or leverage the IIIF APIs.
 
 - [IIIF Image Field](https://www.drupal.org/sandbox/sdellis/2421047) - Drupal 7 module that provides an easy way to add IIIF Images to content types, and configure their display. Supports Image API versions 1.0 or 2.0.
-- [UniversalViewer Omeka S Plugin](https://omeka.org/s/modules/UniversalViewer/) - Universal Viewer is a module for Omeka S that integrates UniversalViewer, a unified online player for any files, so it can display books, images, maps, audio, movies, pdf, 3D, and anything else as long as the appropriate extension is installed. Rotation, zoom, inside search, etc. may be managed too. It uses the resources of any IIIF compliant server.
+- [UniversalViewer Omeka S Module](https://omeka.org/s/modules/UniversalViewer/) - Universal Viewer is a module for Omeka S that integrates UniversalViewer, a unified online player for any files, so it can display books, images, maps, audio, movies, pdf, 3D, and anything else as long as the appropriate extension is installed. Rotation, zoom, inside search, etc. may be managed too. It uses the resources of any IIIF compliant server.
```

Comment:
I'd prefer a short description like this:
Module for Omeka S which integrates UniversalViewer.

Reply:
Shortened description.